src/e2e-test/java/io/cdap/plugin/servicenowsink/actions/ServiceNowSinkPropertiesPageActions.java

```diff
@@ -50,15 +50,15 @@ public class ServiceNowSinkPropertiesPageActions {
   private static Gson gson = new Gson();
 
   public static void getRecordFromServiceNowTable(String query, String tableName)
-    throws OAuthProblemException, OAuthSystemException {
+    throws OAuthProblemException, OAuthSystemException, IOException {
```

Comment:
Just to clarify, this null represents the pageSize?

Reply:
Yes, it got missed in some older change.